modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/transaction/TransactionStorage.scala

```diff
@@ -115,7 +118,8 @@ class TransactionStorage[F[_]: Async: KryoSerializer](
           maybePulled <- maybeWaiting.traverse { waiting =>
             val lastTx = lastAccepted.getOrElse(address, TransactionReference.empty)
             Consecutive
-              .take[F](waiting.toList, lastTx)
+              .take[F](waiting.toList.map(_.signed), lastTx)
```

Comment:
Not sure if modifying take to operate on `Hashed[Transaction]` is not the better idea.